### DISCUSSION TITLE
Smooth animation

### DISCUSSION THREAD

**Discussion by RaZor83** [agronick/aa-torque#8] [Ideas]
Hey, any plan to add the smooth animation back from the original app ?

Thanks for keeping this app updated 👍

**Comment by agronick** [comment]
As far as I know it didn't do anything which is why I removed it. It is always on. It was supposed to tell the gauge to update in 1ms instead of 250ms but it seemed to ignore it. I'm thinking about putting the ability to set the update interval on each item separately and then the animation speed of the gauge would be tied to that.
